content/kips/kip-19.md

```diff
@@ -0,0 +1,83 @@
+---
+kip: 19
+title: Advanced Orders
+status: Draft
+author: Jeremy Chiaramonte
+created: 2022-06-30
+---
+
+## Summary
+
+Advanced orders introduce to new trading strategies for futures.
+
+## Abstract
+
+Leveraging Gelato's keeper system, Kwenta will enable the automation of orders for futures. The first two products to come of this will be Limit Orders & Stop Market Orders. 
+
+## Motivation
+
+Market orders are the only available order type for Kwenta futures at the moment. There are trading benefits by allowing orders to execute in the future based on defined conditions. 
+
+## Specification
+
+Advanced orders are now feasible as a result of "margin accounts" introduced in [KIP-18: Cross Margin](./kip-18.md).
+
+A trader can specify a limit order or stop market order and register the task with Gelato. If the specified condition evaluates true, Gelato keepers will execute the order on behalf of the trader. 
+
+Margin will be committed for an advanced order meaning it cannot be used for any other order or cross margin while the order is active. 
+
+Currently only one order (limit or stop) can be active for a market.
+
+### Slippage
```

Comment:
We might want to be very specific that slippage is paid by the user. These examples are good, however a description of how this comes up would add more clarity.

```
Users specify a price for the execution of their order, however trades are always executed at the current oracle price. Thus, a trader's buy order may be filled at a price below their specified order resulting in positive slippage. Similarly, a trader's stop loss may trigger at a price below their specified order resulting in negative slippage
```

Don't need to add this specifically, but I think an overview is helpful.
```